app/src/main/java/com/github/neighbortrader/foodboardapp/ui/offerOverview/AllOffersModel.java

```diff
@@ -27,13 +30,15 @@ public AllOffersModel(AllOffersController allOffersController) {
     // TODO: doesn't really belong here
```

Comment:
AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH 

Das ist so schnell zu fixen, warum brauch man da ein TODO ?!!???!!!!!111!!!??????11???????????

Reply:
Ist das die literarische Meisterleistung ? :D

Reply:
Ne der letzte kommentar... Trotzdem ernst nehmen

Reply:
Arbeite wenigstens einmal konsequent inhaltlich in den Reviews und lass das Problem an der Stelle absprechen. Damit auch du es verstehst:

Es geht darum das Android dort nicht seinen tatsächlichen Einstiegspunkt hat. Das müsste ich mal herausfinden wo man das macht. Bis dahin FuckeU